mne/viz/utils.py

```diff
@@ -1413,11 +1421,18 @@ def _plot_sensors(pos, colors, bads, ch_names, title, show_names, ax, show,
         ax.set_yticks([])
         fig.subplots_adjust(left=0, bottom=0, right=1, top=1, wspace=None,
                             hspace=None)
-        pos, outlines = _check_outlines(pos, 'head')
+        if to_sphere:
+            pos, outlines = _check_outlines(pos, 'head')
+        else:
+            pos, outlines = _check_outlines(pos, np.array([0.5, 0.5]),
+                                            {'center': (0, 0),
+                                             'scale': (4.5, 4.5)})
         _draw_outlines(ax, outlines)
 
         pts = ax.scatter(pos[:, 0], pos[:, 1], picker=True, c=colors, s=75,
                          edgecolor=edgecolors, linewidth=2)
```

Comment:
Would it be good to add `clip_on=False` in case we allow no normalization at some point?